developer/general/how-to-edit-the-rst-documentation.rst

```diff
@@ -307,148 +307,150 @@ In the following section there is a sample setup to prepare local environments
 for building Qubes OS rST documentation.
 
 
+Creating a Python environment with a dependency management program
+------------------------------------------------------------------
+
+
 Using venv
-----------
+^^^^^^^^^^
 
 
 1. **Install needed packages and clone the repository**
 
-  .. code-block:: console
+   .. code-block:: console
 
-    $ sudo apt install git python3-dev python3.11-venv
-    $ git clone https://github.com/QubesOS/qubes-doc.git
+      $ sudo apt install git python3-dev python3.11-venv
+      $ git clone https://github.com/QubesOS/qubes-doc.git
+      $ cd qubes-doc
 
 2.  **Install Sphinx and Required Extensions**
 
-   Install Sphinx and the necessary extensions (`sphinx-autobuild`, `sphinx-lint`) using `pip`.
+   Enter the virtual environment, install Sphinx and the necessary extensions (:program:`sphinx-autobuild`, :program:`sphinx-lint`).
 
-    .. code-block:: console
+   .. code-block:: console
 
-     $ python -m venv .venv
-     $ source .venv/bin/activate
-     $ pip install -r qubes-doc/requirements.txt
-     $ pip install sphinx sphinx-lint sphinx-autobuild
+      $ python3 -m venv .venv
+      $ . .venv/bin/activate
+      (.venv) $ echo "$VIRTUAL_ENV"
 
+   .. note::
 
-3.  **Verify Installation**
+      You will have to to activate the environment every time a new shell is opened.
 
-    .. code-block:: console
+2.  **Install Sphinx and Required Extensions**
 
-     $ sphinx-build --version
+   .. code-block:: console
 
+      (.venv) $ pip install -r requirements.txt
+      (.venv) $ pip install sphinx sphinx-lint sphinx-autobuild
 
-4.  **Build Documentation**
 
-   Use `sphinx-build` with the `-v` (verbose) flag to generate detailed output during the build process.
+Using poetry
+^^^^^^^^^^^^
 
-    .. code-block:: console
+You can also use `uv <https://docs.astral.sh/uv/getting-started/>`__ if you wish.
```

Comment:
With the rearangement this is somewhat misaligned. I would make a further small section after poetry.